package/CHANGELOG

```diff
@@ -59,6 +59,7 @@ Enhancements
  * Add check and warning for empty (all zero) coordinates in RDKit converter
    (PR #4824)
  * Added `precision` for XYZWriter (Issue #4775, PR #4771) 
+ * Added timestep support for XDR writers and readers(Issue #4905, PR #4908)
```

Comment:
space before (

Reply:
IMPORTANT: moves this (together with your GitHub handle) to the 2.10.0 section!!